accord-core/src/main/java/accord/coordinate/CollectDeps.java

```diff
@@ -0,0 +1,87 @@
+package accord.coordinate;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+import accord.coordinate.tracking.QuorumTracker;
+import accord.local.Node;
+import accord.local.Node.Id;
+import accord.messages.Callback;
+import accord.messages.GetDeps;
+import accord.messages.GetDeps.GetDepsOk;
+import accord.primitives.Deps;
+import accord.primitives.Route;
+import accord.primitives.Timestamp;
+import accord.primitives.Txn;
+import accord.primitives.TxnId;
+import accord.topology.Topologies;
+
+class CollectDeps implements Callback<GetDepsOk>
```

Comment:
Much like `CheckOn`, I intended `Fetch` to be about obtaining a complete representation of the local state, rather than simply obtaining. Here we're just obtaining, which is why I called it `Collect`. I'm open to other names, but would prefer to delineate these things.